source/amazon/services/supported-services/trusted-advisor.rst

```diff
@@ -5,97 +5,130 @@
+`AWS Trusted Advisor <https://aws.amazon.com/premiumsupport/trustedadvisor/>`_ provides real time guidance to help users reduce cost, increase performance, and improve security by optimizing their AWS environment following AWS best practices. Trusted Advisor logs can be stored into a S3 bucket thanks to `Amazon EventBridge <https://aws.amazon.com/eventbridge/>`_ and `Amazon Kinesis Data Firehose <https://aws.amazon.com/kinesis/data-firehose/>`_, allowing Wazuh to process them and generate alerts using the built-in rules Wazuh provides, as well as any :ref:`custom rules <ruleset_custom>` available.
 
 Amazon configuration
 --------------------
 
+Learn how to configure the different services required to integrate Trusted Advisor into Wazuh:
+
+    .. thumbnail:: ../../../images/aws/trusted-advisor.png
+      :align: center
+      :width: 100%
+
+Amazon Kinesis configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Create an Amazon Kinesis Data Firehose delivery stream to be able to store the Trusted Advisor logs into the desired S3 bucket so Wazuh can process them.
+
 #. :ref:`Create a new <S3_bucket>` S3 bucket. (If you want to use an already created one, skip this step).
 
-#. Go to Services > Analytics > Kinesis:
+#. Search for ``kinesis`` in the search bar at the top of the page or go to ``Services`` > ``Analytics`` > ``Kinesis``:
 
-    .. thumbnail:: ../../../images/aws/aws-create-firehose-4.png
+    .. thumbnail:: ../../../images/aws/trusted-kinesis-0.png
       :align: center
-      :width: 70%
+      :width: 100%
 
-    #. If it's the first time you're using this service, you'll see the following screen. Just click on *Get started*:
+#. Select the ``Kinesis Data Firehose`` option and then click in the ``Create delivery stream`` button:
 
-    .. thumbnail:: ../../../images/aws/aws-create-firehose-4.1.png
+    .. thumbnail:: ../../../images/aws/trusted-kinesis-1.png
       :align: center
-      :width: 70%
+      :width: 100%
 
-#. Click on *Create delivery stream* button:
+#. Select ``Direct PUT`` and ``Amazon S3`` as the desired Source and Destination, respectively:
 
-    .. thumbnail:: ../../../images/aws/aws-create-firehose-5.png
+    .. thumbnail:: ../../../images/aws/trusted-kinesis-2.png
       :align: center
-      :width: 70%
+      :width: 100%
 
-#. Put a name to your delivery stream and click on the *Next* button at the bottom of the page:
+#. Choose an appropriate ``Delivery stream name``:
 
-    .. thumbnail:: ../../../images/aws/aws-create-firehose-6.png
+    .. thumbnail:: ../../../images/aws/trusted-kinesis-3.png
       :align: center
-      :width: 70%
+      :width: 100%
 
-#. On the next page, leave both options as *Disabled* and click on *Next*:
+#. Leave both ``Transform`` and ``convert records`` options disabled:
```

Comment:
I think it's clearer to use the name of the fields instead of the first word of the title of the section.
```suggestion
#. Leave both ``Data transformation`` and ``Record format conversion`` options disabled:
```